apisix/plugins/client-control.lua

```diff
@@ -48,7 +48,7 @@ end
 
 
 function _M.rewrite(conf, ctx)
-    if not apisix_ngx_client then
+    if not ok then
```

Comment:
Is this better to show the error message while APISIX is starting?

Reply:
@tokers thx. i guess you mean throw an error when failed calling pcall, so this plugin will not be loaded. I agree with that  actually, but only try to make the patch as small as possbile.

Reply:
The module is loaded by default, and "resty.apisix.client" is only included in APISIX-OpenResty. So showing an error message will annoy people who use regular OpenResty.

Reply:
OK, got it.

Reply:
But from the HTTP semantic, maybe `501` is better in such a case (than `503`).

Reply:
@tokers maybe it's another issue? :)

Reply:
Got it.